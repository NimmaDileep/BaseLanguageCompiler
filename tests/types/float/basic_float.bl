var x: Float = 0.0;

var y: Float = -1.5;

var z: Float = 10.5;

print(x);
print(y);
print(z);