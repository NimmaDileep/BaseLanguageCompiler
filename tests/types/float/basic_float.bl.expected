0.0
-1.5
10.5